grudge/geometry/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+from grudge.geometry.metrics import *
```

Comment:
Link into documentation? That'll also determine the "canonical" import spot for these things.

Reply:
I've updated the geometry module and added documentation throughout. They should be included in the main doc build!